extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzScheduler.java

```diff
@@ -68,13 +67,13 @@
     }
 
     public QuartzScheduler(SchedulerContext context, QuartzSupport quartzSupport, Config config) {
+        Map<String, ScheduledInvoker> invokers;
```

Comment:
Can be moved to where the initialisation occurs? https://github.com/quarkusio/quarkus/pull/9026/files#diff-f3233ae68279acfe2869eaffaeb10682R76

Reply:
Good idea, done